lua/nvim-treesitter/query_predicates.lua

```diff
@@ -1,5 +1,30 @@
 local query = require "vim.treesitter.query"
 
+local html_script_type_languages = {
+  ["importmap"] = "json",
+  ["module"] = "javascript",
+  ["application/ecmascript"] = "javascript",
+  ["text/ecmascript"] = "javascript",
+}
+
+local injection_language_aliases = {
+  ex = "elixir",
+  fc = "func",
+  fir = "firrtl",
+  ha = "hare",
+  hs = "haskell",
+  js = "javascript",
+  kt = "kotlin",
+  ml = "ocaml",
+  pl = "perl",
+  py = "python",
+  ts = "typescript",
+  rs = "rust",
+  sh = "bash",
+  ungram = "ungrammar",
+  uxn = "uxntal",
+}
```

Comment:
shouldn't these be user configurable?

Also the interpretation of injection alias should be upstreamed at some time and maybe even resolve that abreviations by default. It's fine to have them as a experiment here. But when Neovim is trying to import our markdown queries they will need to support that directive in some way or have a more general solution

Reply:
I was thinking this could be a temporary helpful bandaid-like patch until Neovim implements something better.

Reply:
I believe @amaanq 's suggestion was in the right direction, and that they should be configurable as well

So ideally, priority would be

0. Hard coded aliases
1. aliases derived from config in parser package
2. User aliases

I also think that each of those could probably be a subsequent PR

Reply:
yeah, this can also happen in a follow-up PR. I was just thinking of giving the user a chance to manipulate/extend the hardcoded table.

Reply:
Has the feature to allow users to manipulate or extend the hardcoded table been implemented yet?